src/app/pages/search-character/search-character.component.ts

```diff
@@ -15,7 +17,11 @@ export class SearchCharacterComponent {
   charactersList: BasicCharacter[] | null = [];
   characterResultCounter = 0;
 
-  constructor(private readonly apiService: ApiService) {}
+  constructor(
+    private readonly apiService: ApiService,
```

Comment:
- I think it would be a bit more descriptive to use CharactersService as class name
- I know that you did not implement the service, I am only looking for your opinion as an opportunity for improvement
-  @miguelhenaoa What do you think? 
```suggestion
    private readonly charactersService: CharactersService,
```

Reply:
I think it's a good change

Reply:
I agree, is more descriptive.